src/test/java/com/connexta/ingest/IngestITests.java

```diff
@@ -192,8 +195,14 @@ public void afterEach() {
   @Test
   public void testContextLoads() {}
 
-  @Test
-  public void testSuccessfulIngestRequest() throws Exception {
+  @ParameterizedTest
```

Comment:
✏️ Push these parameterized tests down to `IngestControllerTests`. We don't need to run itests just to test different date formats. I think `testSuccessfulIngestRequest` would be a good place to add this parameter.